### ISSUE TITLE
Edge v80 is not being loaded.

### ISSUE THREAD

**Issue by schloerke** [bonigarcia/webdrivermanager#457]
**Description of the problem**: Edge driver does not seem to listen to the version supplied. Ex `80.0.361.69`.

Possibly related: https://github.com/bonigarcia/webdrivermanager/issues/444
(Internally related: https://github.com/schloerke/shinyjster/issues/31)

**Browser and version**: Windows Edge 80

**Operating system**: GitHub Actions - Windows Server 2019

**WebDriverManager version**: 3.8.1

**WebDriverManager use**: `WebDriverManager.edgedriver().version("80.0.361.69").setup()`

**WebDriverManager traces**: 

I couldn't figure out how to get the logging to work. 😞 

I was able to ask the `WebDriverManager` which versions were available, 

```kotlin
println(WebDriverManager.getInstance(driverType).getVersions().joinToString())
// 1.10240, 2.10586, 3.14393, 4.15063, 5.16299, 6.17134, 81.0.416.34, 81.0.416.41, 81.0.416.45, 82.0.457.0, 82.0.458.0, 82.0.459.1, 83.0.473.0, 83.0.474.0, 83.0.477.0
```

Even when setting a known version, it was still not respected.

```kotlin
WebDriverManager.edgedriver().version("81.0.416.34").setup()
// ...

// output after asking for a website
// Starting MSEdgeDriver 83.0.477.0 (03c3105e4ebf9b7de71097cf20125f4d7ed1f72d) on port 53264 
```

What I find odd is that the GitHub Actions images (currently) support Edge v80 ([Browser](https://github.com/actions/virtual-environments/blob/master/images/win/Windows2019-Readme.md#microsoft-edge), [Driver](https://github.com/actions/virtual-environments/blob/master/images/win/Windows2019-Readme.md#microsoft-edge-driver)).  For my knowledge, Edge v83 is [not installed on GitHub Actions images](https://github.com/actions/virtual-environments/blob/5070020018246c0bd71d650b17b1d8ec0f25d6bc/images/win/scripts/Installers/Install-Edge.ps1#L6).  

I don't know why Edge v83 is being set when the installed browser is Edge v80 which you have set in [your `versions.properties` file](https://github.com/bonigarcia/webdrivermanager/blob/master/src/main/resources/versions.properties#L78).

**Error trace**: 

```
Starting MSEdgeDriver 83.0.477.0 (03c3105e4ebf9b7de71097cf20125f4d7ed1f72d) on port 53264 
Only local connections are allowed. Please see https://chromedriver.chromium.org/security-considerations for suggestions on keeping MSEdgeDriver safe. MSEdgeDriver was started successfully. 
Exception in thread "main" java.lang.reflect.InvocationTargetException
 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
 	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
 	at com.rstudio.seleniumRunner.MainKt.main(Main.kt:87) Caused by: org.openqa.selenium.SessionNotCreatedException: session not created: This version of MSEdgeDriver only supports MSEdge version 83 Build info: version: 'unknown', revision: 'unknown', time: 'unknown' System info: host: 'fv-az27', ip: '10.1.0.4', os.name: 'Windows Server 2019', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_222' Driver info: driver.version: EdgeDriver remote stacktrace: Backtrace:
 	Ordinal0 [0x00007FF6F32FB262+3846754]
 	Ordinal0 [0x00007FF6F31EB459+2733145]
 	Ordinal0 [0x00007FF6F307C07C+1228924]
 	Ordinal0 [0x00007FF6F2FD4085+540805]
 	Ordinal0 [0x00007FF6F2FCF198+520600]
 	Ordinal0 [0x00007FF6F2FCAA61+502369]
 	Ordinal0 [0x00007FF6F2FFF365+717669]
 	Ordinal0 [0x00007FF6F2FFC73F+706367]
 	Ordinal0 [0x00007FF6F2FD65C3+550339]
 	Ordinal0 [0x00007FF6F2FD738E+553870]
 	Ordinal0 [0x00007FF6F3208745+2852677]
 	GetHandleVerifier [0x00007FF6F3437B6A+1162954]
 	GetHandleVerifier [0x00007FF6F344C671+1247697]
 	GetHandleVerifier [0x00007FF6F34384C4+1165348]
 	Ordinal0 [0x00007FF6F31FC1D8+2802136]
 	Ordinal0 [0x00007FF6F320A4B9+2860217]
 	Ordinal0 [0x00007FF6F3228F18+2985752]
 	BaseThreadInitThunk [0x00007FFC302A7974+20]
 	RtlUserThreadStart [0x00007FFC3040A261+33]
  	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
 	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
 	at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
 	at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
 	at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
 	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
 	at java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:958)
 	at java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:126)
 	at java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:499)
 	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:486)
 	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472)
 	at java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:152)
 	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
 	at java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:531)
 	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128) 	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:74) 
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
 	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
 	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
 	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
 	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
 	at org.openqa.selenium.edge.EdgeDriver.<init>(EdgeDriver.java:141)
 	at org.openqa.selenium.edge.EdgeDriver.<init>(EdgeDriver.java:96) 	... 5 more 
```

**Comment by bonigarcia** [comment]
This is a limitation of the current manager for Edge. I plan to solve it and release a new version ASAP.

**Comment by eliasnogueira** [comment]
I could successfully run a test with `80.0.361.69` version locally from the master branch.

```
2020-04-28 17:42:07 [main] INFO  i.g.bonigarcia.wdm.WebDriverManager.getDriversFromXml(1041) -- Reading https://msedgedriver.azureedge.net/?restype=container&comp=list to seek msedgedriver
2020-04-28 17:42:17 [main] INFO  io.github.bonigarcia.wdm.Downloader.downloadAndExtract(125) -- Downloading https://msedgedriver.azureedge.net/80.0.361.69/edgedriver_mac64.zip
2020-04-28 17:42:29 [main] INFO  io.github.bonigarcia.wdm.Downloader.extract(179) -- Extracting binary from compressed file edgedriver_mac64.zip
2020-04-28 17:42:30 [main] INFO  i.g.bonigarcia.wdm.WebDriverManager.exportDriver(563) -- Exporting webdriver.edge.driver as /Users/eliasnogueira/.m2/repository/webdriver/edgedriver/mac64/x64/80.0.361.69/msedgedriver
Starting MSEdgeDriver 80.0.361.69 (11b2f9c770113c9712ab7dffaca9ea3596d0deb3) on port 4305
Only local connections are allowed.
Please protect ports used by MSEdgeDriver and related test frameworks to prevent access by malicious code.
OneAuth Version 0.16.0 (com.microsoft.Edge)
[1588088551.839][WARNING]: This version of MSEdgeDriver has not been tested with MSEdge version 81.
Apr 28, 2020 5:42:31 PM org.openqa.selenium.remote.ProtocolHandshake createSession
INFO: Detected dialect: W3C
[1588088552.269][SEVERE]: Timed out receiving message from renderer: 0.100
[1588088552.370][SEVERE]: Timed out receiving message from renderer: 0.100
[1588088552.471][SEVERE]: Timed out receiving message from renderer: 0.100
[1588088552.573][SEVERE]: Timed out receiving message from renderer: 0.100
[1588088552.920][SEVERE]: Timed out receiving message from renderer: 0.100
[1588088553.021][SEVERE]: Timed out receiving message from renderer: 0.100
[1588088553.123][SEVERE]: Timed out receiving message from renderer: 0.100
[1588088553.277][SEVERE]: Timed out receiving message from renderer: 0.100

Process finished with exit code 0
```

**Comment by bonigarcia** [comment]
This feature is available in WebDriverManager 4.0.0, recently released.
